tests/unit/test_charm.py

```diff
@@ -113,10 +113,16 @@ def test_topology_added_if_user_provided_config_with_group_by(self, *unused):
     def test_charm_blocks_if_user_provided_config_with_templates(self, *unused):
         new_config = yaml.dump({"templates": ["/what/ever/*.tmpl"]})
         self.harness.update_config({"config_file": new_config})
+        self.harness.evaluate_status()
         self.assertIsInstance(self.harness.charm.unit.status, BlockedStatus)
 
+        # Manual cleanup, due to https://github.com/canonical/operator/issues/736
+        self.harness.charm.unit._collected_statuses.clear()
```

Comment:
Presumably this part of it (and similar lines below) will be fixed by https://github.com/canonical/operator/pull/1048?

Reply:
Yes!